app/ui/browser/views/browser.jsx

```diff
@@ -82,8 +82,9 @@ class BrowserWindow extends Component {
       }));
     };
     const onLocationContextMenu = e => menuLocationContext(e.target, currentPageId, dispatch);
+    const onClearCompletions = () => dispatch(actions.clearCompletions());
     const onLocationReset = () => {
-      dispatch(actions.setUserTypedLocation(currentPageId, { text: void 0 }));
+      dispatch(actions.locationChanged(currentPageId, { text: null }));
```

Comment:
It looks like this is an existing action. Do we know if there was a reason this wasn't already being used here?